.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -6,29 +6,56 @@ labels: type/bug
 assignees: ''
 
 ---
+<!-- 
+Welcome to Airbyte project!
+We're really glad with your contribution this will help us build 
+an amazing tool. We accept contributions too! You can find them filtering the
+issues by good-first-issues or maybe... try to solve this one? 
+(we could help you debug and pointing the path)
```

Comment:
```suggestion
We're really appreciate your report and know that this will help us build an amazing tool. If you want to contribute yourself, you can find a good place to start by searching for the good-first-issues label or maybe... by trying to solve this one? (we can help debug this with you!)
```